src/app/Helpers/Stats.php

```diff
@@ -23,12 +23,100 @@ public static function countFutureAvailability() {
       return Location::available()->count();
     }
 
+    /**
+     * Get the number of locations that have never had their information
+     * updated.
+     */
+    public static function countLocationsNeverUpdated() {
+        return count(DB::select(DB::raw("select id from locations where id NOT IN (select location_id from availabilities group by location_id)")));
```

Comment:
Instead of a `group by` you could also do `select distinct location_id`. I believe your method works, too.